packages/paste-codemods/tools/create-package-mappings.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const {getRepoPackages} = require('../../../tools/utils/getRepoPackages');
+const {writeToFile} = require('../../../tools/utils/writeToFile');
+
+async function generatePackageExportsMap() {
+  // Object to store all the generated mappings for our codemod
+  const mapping = {};
+
+  // Get all Paste packages
+  const allPastePackages = await getRepoPackages();
+
+  // For each package in Paste:
+  allPastePackages.forEach(({name, location}) => {
+    // We don't want to use the 'form' or 'typography' packages because they are deprecated
+    // Skip them for now
+    if (name === '@twilio-paste/typography' || name === '@twilio-paste/form') return;
```

Comment:
This would probably be better if you had an array of deprecated packages and checked if they were in that array instead of an if statement with or's for maintenance but such an easy change I probably wouldn't worry about it

Reply:
Totes. Done